fastlane_core/lib/fastlane_core/cert_checker.rb

```diff
@@ -139,30 +139,68 @@ def self.install_missing_wwdr_certificates
       # Install all Worldwide Developer Relations Intermediate Certificates listed here: https://www.apple.com/certificateauthority/
       missing = WWDRCA_CERTIFICATES.map { |c| c[:alias] } - installed_wwdr_certificates
       missing.each do |cert_alias|
-        install_wwdr_certificate(cert_alias)
+        Tempfile.create(File.basename('fastlane-wwdr-cert-')) do |tmpfile|
```

Comment:
This is not how `Tempfile.create` is supposed to be used.

The parameter to `Tempfile.create` is supposed to be a **string that will be used as a prefix** for the random filename that `Tempfile` will generate to guarantee uniqueness of that temporary file on disk.

So using `File.basename` in that context for that argument doesn't really make sense (I'm not sure what the goal of that `File.basename` was here given the string you pass is already a hardcoded string literal that is free of any `/` path separator anyway 🙃

Reply:
> This is not how `Tempfile.create` is supposed to be used.
> 
> The parameter to `Tempfile.create` is supposed to be a **string that will be used as a prefix** for the random filename that `Tempfile` will generate to guarantee uniqueness of that temporary file on disk.
> 
> So using `File.basename` in that context for that argument doesn't really make sense (I'm not sure what the goal of that `File.basename` was here given the string you pass is already a hardcoded string literal that is free of any `/` path separator anyway 🙃

Damn... sloppy leftover, from when I was thinking that the file wasn't created properly.